src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java

```diff
@@ -675,6 +676,15 @@ public void onPlayerCommandPreprocess(PlayerCommandPreprocessEvent event)
             event.setCancelled(true);
             return;
         }
+        
+        // Makes it so that imposters can't jump when frozen
+        if (!TFM_AdminList.isSuperAdmin(player) && playerdata.isFrozen())
```

Comment:
Should be if (TFM_AdminList.IsAdminImposter(player) && playerdata.isFrozen()) 
i think.